packages/experiments/src/components/Announced/examples/Announced.Asynchronous.Example.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import { Announced } from '../Announced';
+import { createArray, createRef } from 'office-ui-fabric-react/lib/Utilities';
+import { Image } from 'office-ui-fabric-react/lib/Image';
+import { FocusZone } from 'office-ui-fabric-react/lib/FocusZone';
+import { Toggle } from 'office-ui-fabric-react/lib/Toggle';
+import './Announced.Example.scss';
+import { ProgressIndicator } from 'office-ui-fabric-react/lib/ProgressIndicator';
+import { VerticalStack } from '../../Stack';
+
+export interface IAnnouncedAsynchronousExampleState {
+  photos: { url: string; width: number; height: number }[];
+  total: number;
+  seconds: number;
+  announced: JSX.Element | undefined;
+  percentComplete: number;
+  loading: boolean;
+}
+
+export interface IAnnouncedAsynchronousExampleProps {}
+
+export class AnnouncedAsynchronousExample extends React.Component<IAnnouncedAsynchronousExampleProps, IAnnouncedAsynchronousExampleState> {
+  private _root = createRef<HTMLElement>();
+  private timer: number;
+  private increaseTotal: number;
+  private delay: number;
+
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      photos: this._createPhotos(),
+      total: 0,
+      seconds: 0,
+      announced: undefined,
+      percentComplete: 0,
+      loading: false
+    };
+
+    this._renderPhotos = this._renderPhotos.bind(this);
+    this._renderAnnounced = this._renderAnnounced.bind(this);
+    this._onFocusPhotoCell = this._onFocusPhotoCell.bind(this);
+    this._onToggleChange = this._onToggleChange.bind(this);
+
+    this.increaseTotal = setInterval(() => {
+      if (this.state.loading && this.state.total < this.state.photos.length) {
+        this.setState({ total: this.state.total + 1 });
+      }
+    }, 2000);
+
+    this.timer = setInterval(() => {
+      if (this.state.loading) {
+        this.setState({ seconds: this.state.seconds + 1 });
+
+        if (this.state.seconds % this.delay === 0) {
+          this.setState({
+            announced: (
+              <Announced message={`${this.state.total}/${this.state.photos.length} photos loaded`} id={'announced-' + this.state.total} />
+            )
+          });
+        } else {
+          this.setState({ announced: undefined });
+        }
+      }
+    }, 1000);
+
+    this.delay = 10;
+  }
+
+  public render(): JSX.Element {
+    const { percentComplete } = this.state;
+
+    return (
+      <VerticalStack gap={10}>
+        <div>
```

Comment:
nit: I'd consider making these `<p>` tags rather than `<div>` for semantic reasons unless it involves layout changes.